src/styles/Navbar.css

```diff
@@ -136,3 +135,26 @@
 		}
 	}
 }
+
+@media (max-width: 400px) {
+	.navbar {
+		padding: 0.75rem 0.75rem;
+		height: 15vw;
+	}
+
+	.nav-hack-logo {
+		height: 8vw;
+	}
+
+	.nav-hack h1 {
+		font-size: 6vw;
+	}
+
+	.hamburger {
+		transform: scale(0.8);
+	}
+
+	.navbar-links.active {
```

Comment:
Another issue I'm seeing on the deploy is that pages still aren't fully responsive on mobile, and we can still scroll so they become off the page.
<img width="395" alt="image" src="https://github.com/user-attachments/assets/a401e953-0cf2-43fa-9f62-f15b28d7c286" />

Apologies if the issue description was unclear, could you also make these other pages responsive as well? I think this particular issue should be to make all pages responsive.